2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Configure stats pid to 108490
2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Loading settings from /home/hghallab/.config/wandb/settings
2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Loading settings from /home/hghallab/Dvlp/GANs/DCGAN/wandb/settings
2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-30 12:22:25,171 INFO    MainThread:108490 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-30 12:22:25,172 INFO    MainThread:108490 [wandb_init.py:_log_setup():521] Logging user logs to /home/hghallab/Dvlp/GANs/DCGAN/wandb/run-20240430_122225-9f20qx9z/logs/debug.log
2024-04-30 12:22:25,172 INFO    MainThread:108490 [wandb_init.py:_log_setup():522] Logging internal logs to /home/hghallab/Dvlp/GANs/DCGAN/wandb/run-20240430_122225-9f20qx9z/logs/debug-internal.log
2024-04-30 12:22:25,172 INFO    MainThread:108490 [wandb_init.py:init():561] calling init triggers
2024-04-30 12:22:25,172 INFO    MainThread:108490 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'seed': 12, 'debug': False, 'model_name': 'DCGANsmall|templates', 'loss_function': 'BCELoss', 'train_bs': 8, 'valid_bs': 8, 'img_size': [64, 64], 'epochs': 500, 'lr': 0.0002, 'n_accumulate': 4, 'device': device(type='cpu')}
2024-04-30 12:22:25,172 INFO    MainThread:108490 [wandb_init.py:init():586] re-initializing run, found existing run on stack: rw5skurh
2024-04-30 12:22:25,174 INFO    MainThread:108490 [wandb_run.py:_finish():2064] finishing run houda222/DCGAN/rw5skurh
2024-04-30 12:22:25,175 ERROR   MainThread:108490 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-30 12:22:25,175 INFO    MainThread:108490 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-30 12:22:25,175 INFO    MainThread:108490 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-30 12:22:25,175 INFO    MainThread:108490 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-30 12:22:25,175 INFO    MainThread:108490 [wandb_run.py:_restore():2287] restore
2024-04-30 12:22:25,175 INFO    MainThread:108490 [wandb_run.py:_restore():2293] restore done
2024-04-30 12:23:41,610 INFO    MainThread:108490 [wandb_init.py:init():611] starting backend
2024-04-30 12:23:41,610 INFO    MainThread:108490 [wandb_init.py:init():615] setting up manager
2024-04-30 12:23:41,611 INFO    MainThread:108490 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-30 12:23:41,611 INFO    MainThread:108490 [wandb_init.py:init():623] backend started and connected
2024-04-30 12:23:41,613 INFO    MainThread:108490 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-30 12:23:41,613 INFO    MainThread:108490 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-30 12:23:41,613 INFO    MainThread:108490 [wandb_init.py:init():715] updated telemetry
2024-04-30 12:23:41,617 INFO    MainThread:108490 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
